DuckDuckGo/AppRatingPrompt.swift

```diff
@@ -95,14 +101,20 @@ class AppRatingPromptCoreDataStorage: AppRatingPromptStorage {
     
     public init() { }
     
-    func ratingPromptEntity() -> AppRatingPromptEntity {
+    func ratingPromptEntity() -> AppRatingPromptEntity? {
 
         let fetchRequest: NSFetchRequest<AppRatingPromptEntity> = AppRatingPromptEntity.fetchRequest()
-        
-        guard let results = try? context.fetch(fetchRequest) else {
-            fatalError("Error fetching AppRatingPromptEntity")
+
+        let results: [AppRatingPromptEntity]
+
+        do {
+            results = try context.fetch(fetchRequest)
+        } catch {
+            DailyPixel.fireDailyAndCount(pixel: .appRatingPromptFetchError, error: error)
```

Comment:
Does it send atb and appVersion? It should not.

Reply:
Oh I missed this comment. ATB is a good point but thought we’d perhaps want to know when it’s fixed by using appVersion. It’s included in the privacy triage.